cpp/ql/src/change-notes/2024-07-10-newarrayexpr-arraysize.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
The change not should go in `cpp/ql/lib/change-notes` (it's a library change, not a query change). That should also fix the error you're getting now.